src/python/pants/backend/typescript/README.md

```diff
@@ -1,33 +1,209 @@
-# TypeScript Backend - Under Development!
+# TypeScript Backend
 
-The TypeScript backend is incomplete because it is under active development! Thus, it
-may not be used for any real TypeScript projects in production.
+## Limitations
 
-## Functionality
+### Install without resolve / Install from different resolve
 
-After enabling the `"pants.backend.experimental.typescript"` backend in the `pants.toml` file, you can:
+PR_NOTE: Needs discussion. These options currently aren't supported. 
```

Comment:
'options' - I think this comment was referring to the tool subsystem options - version and install from resolve. I couldn't see a way to disable these - I'm not sure how much it matters. 
Per comment continued below - I can't see that they would practically be needed - but let me know.